release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1446,6 +1475,19 @@ In addition, the following commands related to the format have been removed from
 *Web console (Administrator perspective)*
 
 //section 1
+* The *Jobs* table *Completions* column was sorting on the number of *Desired* completions instead of the number of *Succeeded* completions. The data is presented as *# Succeeded of # Desired*, so when sorting by that column the results looked confusing because the data was sorted by the second number. The *Jobs* *Completions* column now sorts on the *# Succeeded* for better understanding. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1902003[*BZ#1902003*])
+
+* Parsing undefined values results in a not a number (NaN) exception and the *Chart* tooltip shows a box with no values. Specify a start date when fetching data to ensure that the results are synced and enure undefined values are not parsed so that the *Chart* toolip shows correct values. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1906304[*BZ#1906304*])
+
+* The input labels in the *Manage Columns* modal were not clickable buttons, so you could not click them to manage the columns. With this bug fix, the labels are now clickable buttons that you can use to manage columns. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1908343[*BZ#1908343*])
+
+* CSI provisioners were not listed when creating a storage class on the Google Cloud Platform. With this bug fix, the issue is resolved. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1910500[*BZ#1910500*])
+
+* Previously if when using the web console, the user clicks into a *Cluster Role* from *User Management -> Roles* list view, the back link from the details page is *Cluster Roles* which provides a generic list view of *Cluster Roles*. This caused backward web console navigation to redirect to the incorrect page. The fix is for the back link to direct the user to the *Role/Bindings* list view from the *Cluster Role/RoleBinding* details page. This allows the user to correctly navigate backward in the web console.  (link:https://bugzilla.redhat.com/show_bug.cgi?id=1915971[*BZ#1915971*])
+
+* Previously, *Created date time* was not displayed in a format that a person could read. This meant it was difficult to understand and use the time shown in UTC. The fix was to reformat the displayed time so that UTC was readable and understandable. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917241[*BZ#1917241*])
```

Comment:
```suggestion
* Previously, *Created date time* was not displayed in a format that a person could read, which made it difficult to understand and use the time shown in UTC. With this release, the displayed time is reformatted so that UTC is readable and understandable. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1917241[*BZ#1917241*])
```